ktor-client/ktor-client-plugins/ktor-client-websockets/common/test/io/ktor/client/tests/plugins/WebSocketRemoteTest.kt

```diff
@@ -119,23 +119,29 @@ class WebSocketRemoteTest : ClientLoader() {
     }
 
     private class CustomException : Exception()
-
+    
```

Comment:
Please fix the formatting here

Reply:
Thanks for the comment, I've fixed formatting